src/features/client-dashboard/components/ClientActionRequiredWidget.tsx

```
@@ -0,0 +1,90 @@
+    </header>
+    <div className="flex-1 overflow-y-auto px-5 py-4">
+      {loading ? (
+        <div className="flex items-center justify-center py-8">
+          <LoadingSpinner size="md" />
+        </div>
+      ) : error ? (
+        <div className="rounded-xl border border-card-border bg-card px-3 py-2 text-sm text-input-text">
+          {error}
+        </div>
+      ) : items.length === 0 ? (
+        <p className="text-sm text-input-placeholder">You're all caught up. No actions waiting on you right now.</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)